apps/frontend/src/pages/SingleNeighborhoodPage/SingleNeighborhoodPage.tsx

```diff
@@ -39,7 +39,17 @@ export async function action({ params, request }: ActionFunctionArgs) {
   const requestData = Object.fromEntries(formData) as unknown as RequestData;
   requestData.neighborhoodId = Number(params.id);
 
-  const response = await requestServices.createRequest(requestData);
+  const intent = formData.get('intent');
```

Comment:
Good thinking with this!

Reply:
Thanks, I got it from the react-router docs